kohese-portal/client/src/services/item-repository/item-repository.service.ts

```diff
@@ -757,8 +757,9 @@ export class ItemRepository {
 	        0].propertyName] + '](' + window.location.origin + LocationMap[
 	        'Explore'].route + ';id=' + koheseObject.id + ')\n\n');
   	  } else {
-	      representation += (koheseObject[formatDefinition.header.contents[0].
-	        propertyName] + '\n\n');
+        if(formatDefinitionType === FormatDefinitionType.DOCUMENT) {
```

Comment:
Name is still showing without label on Document tab.

Reply:
we went with a different implementation that satisfies the AC